integration/testproject/outfile

```diff
@@ -4,5 +4,12 @@ src/cities.ts:1 - sepehub
 src/cities.ts:2 - kuariob
```

Comment:
I've removed this file from master as it's generated. I think you should modify the `outfile.base` instead.

Reply:
👍

Reply:
(I still see this file in `master` — do you want me to remove it?)